### PR TITLE
Dump JSON

### PR DESCRIPTION
**Pull request by Avielyo10**
The current dump is pseudo YAML which makes it hard to parse.

### CODE REVIEW

**Comment by puiterwijk** [comment]
I would very much prefer this to be a flag, or to use YAML: the `dump-` commands are primarily intended for manual use. And while I'm not opposed to making it machine-parseable, I don't want to lose the ability to read it by hand.

**Comment by puiterwijk** [comment]
Additionally, if you change one, please also change the other `dump-` command.

**Comment by Avielyo10** [comment]
@puiterwijk Ok, I'll
